tests/test_100_temp.py

```diff
@@ -0,0 +1,422 @@
+import copy
```

Comment:
This file should have a better name, what is this supposed to test?

Reply:
It was/is a placeholder for tests to come.
We can remove it for now.

Reply:
I'll simply rename it

Reply:
pruned here
https://github.com/IdentityPython/oidc-op/pull/155/commits/accc68d26c1add00adde6c474eb1cfe256b06e40